.circleci/config.yml

```diff
@@ -420,6 +253,7 @@ workflows:
                         - develop
       - deploy_test_build:
```

Comment:
![image](https://user-images.githubusercontent.com/29925144/49712804-3c5ecc00-fc57-11e8-8e3b-fb68add01694.png)
This may not happen on the master or staging branches as the develop branch is the only one picked